test/integration_test.js

```diff
@@ -33,9 +33,6 @@ describe("Integration tests", function() {
   this.timeout(0);
 
   before(() => {
-    if (skipLocalServer) {
-      return;
-    }
```

Comment:
I think that's the right move, but we should expose tracebacks when they occur in the test server.